cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -1320,7 +1320,16 @@ data:
                 "forecastingEnabled": "{{ template "forecastingEnabled" . }}",
                 "chartVersion": "DEVELOP_BRANCH",
                 "hourlyDataRetention": "{{ (.Values.kubecostModel.etlHourlyStoreDurationHours) }}",
-                "dailyDataRetention": "{{ (.Values.kubecostModel.etlDailyStoreDurationDays) }}"
+                "dailyDataRetention": "{{ (.Values.kubecostModel.etlDailyStoreDurationDays) }}",
```

Comment:
Are we using these values in the frontend yet? If not, this could be a good time to transition to `.Values.kubecostAggregator.etlDailyStoreDurationDays`?

Reply:
I'll add it. I don't see the hourly Store in aggregator though. Am I blind?

Reply:
we are targeting 2.4.0, but FE is almost ready for hosted.

Reply:
@jessegoodier `ETL_HOURLY_STORE_DURATION_HOURS` is an environment variable that Aggregator accepts, but we are not setting anywhere in the Helm chart. I believe the backend will default to `49`